File: scripts/cs/shard3d-csscript-core64/Core/ECSMgr.cs
namespace Shard3D.Core
{
    public class ECS
    {
        public static Actor SpawnActor(string name = "Some kind of Actor")
        {
            InternalCalls.SpawnActor(out ulong guid, name);
            return new Actor(guid);
        }
        public static Actor GetActor(ulong guid)
        {
            return new Actor(guid);
        }
        public static Actor GetActor(string tag)
        {
            InternalCalls.GetActorByTag(out ulong guid, tag);
            return new Actor(guid);
        }
        public static void KillActor(ulong guid)
        {
            InternalCalls.KillActor(guid);
        }
    }
}

File: editor/scripting/vb/shard3d-vbscript-core64/Core/ECSMgr.vb
Namespace Shard3D.Core
    Public Class ECS
        Public Shared Function SpawnActor(Optional ByVal name As String = "Some kind of Actor") As Actor
            Dim guid As ULong = Nothing
            InternalCalls.SpawnActor(guid, name)
            Return New Actor(guid)
        End Function
    End Class
End Namespace